glommio/src/sys/uring.rs

```diff
@@ -1258,7 +1259,7 @@ impl Reactor {
             latency_ring: RefCell::new(latency_ring),
             poll_ring: RefCell::new(poll_ring),
             timeout_src: Cell::new(None),
-            syscall_thread: BlockingThread::new(notifier.clone()),
+            blocking_thread: BlockingThreadPool::new(PoolPlacement::Unbound(1), notifier.clone())?,
```

Comment:
why 1?
I see no reason not to distribute threads into many cpus.

Reply:
You're looking at the commit that implements the pool itself. A way to configures the poll will come in a further commit :)